### ISSUE TITLE
replace changelog parser with new CPAN::Changes

### ISSUE THREAD

**Issue by haarg** [metacpan/metacpan-web#1875]
I have a new CPAN::Changes nearly complete, and it should be able to replace the parsing routine we are currently using once it is released.

**Comment by oalders** [comment]
@haarg what's the status of this?

**Comment by oalders** [comment]
@haarg is this complete?

**Comment by haarg** [comment]
No, we're still using our own parser. I did release the new CPAN::Changes which uses a parser based on the same code we're using, but there are some differences between the two.
